### DISCUSSION TITLE
Instead of using SBT, I want to switch to Nft Collection.

### DISCUSSION THREAD

**Discussion by brent0126** [ton-org/game-engines-sdk#15] [Q&A]
Instead of using SBT, I want to switch to NFT Collection. How would that work, and I want to be able to buy and sell those NFTs using Jetton.

**Comment by barinbritva** [comment]
@brent0126 hi,

Yes, you can do it. You can use `game-engines-sdk` and `assets-sdk` to implement any custom logic

**Reply by brent0126** [reply]
Thanks. Can you give me some guidance on this?

**Reply by barinbritva** [reply]
I'm not sure what you mean. The SDK provides methods to work with bot `nfts` and `sbts`. You just need to use `openNftCollection` instead of `openSbtCollection` in that place where you need to use `nft` instead of `sbt`

Do you have an code example to me be able to understand what do you do?
